Cant't resolve fs after updating to 13.0.3 version

I am using file-type in an angular app. When running ng serve, I get the following:
ERROR in ./node_modules/strtok3/lib/FsPromise.js
Module not found: Error: Can't resolve 'fs' in /node_modules/strtok3/lib'
With v12 it works

--- a/core.js
+++ b/core.js
@@ -57,6 +57,16 @@ function _check(buffer, headers, options) {
 }
 
 async function fromTokenizer(tokenizer) {
+	try {
+		return _fromTokenizer(tokenizer);
+	} catch (error) {
+		if (!(error instanceof strtok3.EndOfStreamError)) {
+			throw error;
+		}
+	}
+}
+
+async function _fromTokenizer(tokenizer) {
 	let buffer = Buffer.alloc(minimumBytes);
 	const bytesRead = 12;
 	const check = (header, options) => _check(buffer, header, options);

